network_simulator/device.py

```diff
@@ -0,0 +1,79 @@
+"""This module contains all device definitions.
+
+.. autosummary::
+    Host
+    Router
+"""
+
+# Imports go here.
+# e.g.    from .common import ACK_PACKET_SIZE_BITS
+
+class Device(object):
+    """Representation of a device (Host or Router).
+
+    :param str name: name of this device from the input file.
+    :param str address: IP address of this device.
+    :ivar str name: name of this device from the input file.
```

Comment:
"name" is redundant. Also, "address" is just an address, not an IP address (it could be "H1", "H2", etc).